controllers/student.js

```diff
@@ -0,0 +1,67 @@
+import { sendResponse, sendErrorResponse } from '../utils/response.js';
+import { createStudent, getAllStudents, getStudentById, updateStudent, deleteStudent } from '../services/student.js';
+import studentValidationSchema from '../utils/studentValidation.js';
+import { HTTP_STATUS_CODES } from '../utils/statusCodes.js';  // Import status codes
+
+export const createStudentController = async (req, res) => {
+  try {
+    const { error } = studentValidationSchema.validate(req.body);
+    if (error) {
+      return sendErrorResponse(res, error.details[0].message, HTTP_STATUS_CODES.BAD_REQUEST);
+    }
+    const student = await createStudent(req.body);
+    sendResponse(res, HTTP_STATUS_CODES.CREATED, 'Student created successfully', student);
+  } catch (error) {
+    sendErrorResponse(res, error.message || 'Error creating student', HTTP_STATUS_CODES.INTERNAL_SERVER_ERROR);
```

Comment:
```suggestion
    sendErrorResponse(res, error.message || 'An error occurred while creating student, HTTP_STATUS_CODES.INTERNAL_SERVER_ERROR);
```

Add meaningful **default error** message